java/src/jmri/jmrit/logix/WarrantShutdownTask.java

```diff
@@ -83,7 +83,7 @@ private boolean makeMergeCandidates() {
             if (anomaly.size() > 0) {
                 _anomalies.put(entry.getKey(), anomaly);
             }
-            _mergeCandidates.put(entry.getKey(), true);
+            _mergeCandidates.put(entry.getKey(), Boolean.valueOf(true));
```

Comment:
Unless this was flagged by FindBugs or something else, boxing like this is generally not needed when storing boolean values in a HashMap or HashTable (this is new in Java 7 or 8, I forget which).